.expeditor/scripts/release_habitat/publish_docker_studio.ps1

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env powershell
+
+#Requires -Version 5
+
+. $PSScriptRoot/shared.ps1
+
+$ErrorActionPreference="stop"
+$ReleaseChannel=Get-ReleaseChannel
+
+Write-Host "--- Building the Windows Docker Studio"
+
+& $PSScriptRoot/build_docker_image.ps1 -ReleaseChannel $ReleaseChannel -BldrUrl $Env:ACCEPTANCE_HAB_BLDR_URL
```

Comment:
Acceptance again

Reply:
I think some of these container image creation scripts could stand for some consolidation and cleaning up, but I wanted to keep the changes in this PR pretty minimal. I'm just focused here on getting the existing code into the new pipeline framework.